src/style.css

```diff
@@ -0,0 +1,71 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
```

Comment:
don't use so many properties for * selector, only box-sizing from this list is okay here, others too heavy